### PR TITLE
[scripts/get] makes sudo an optional dependency

### PR DESCRIPTION
**Pull request by tonyfahrion**
If the execution user is already the root user, this avoids requiring the 
sudo command to be installed within the environment.

Which might help in CI environments like alpine.

Tested with `ubuntu:latest` and `alpine:latest`

### CODE REVIEW

File: scripts/get
Comment (thomastaylor312) [inline scripts/get]: If I remember my bash correctly, I think the best way to do this is to check if `$EUID` is equal to 0. According to [this SO answer](https://stackoverflow.com/a/18216122), `EUID` is read only and the best way to make sure it is a root user.

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
This is a great patch! I just ran into this the other day. One change is needed on how you are checking for root and then it should be good

File: scripts/get
Comment (twolfy) [inline scripts/get]: Thx! Very good point! Didn't know that `$EUID` exists (`env` doesn't show it).

I changed the PR via 8eace38, please have a look

**Review by technosophos — APPROVED** [review]
LGTM

**Comment by technosophos** [comment]
I'll let @thomastaylor312 make the final call on it, but this is looking good.

**Comment by technosophos** [comment]
Also, IMHO this meets the "stability/security" requirement to be added to 2.6.

**Comment by thomastaylor312** [comment]
@technosophos I second that opinion. Though technically this will be available as soon as we merge because the install instructions `curl` the script from master 🙂

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**
